mms/scala/README.md

```diff
@@ -0,0 +1,28 @@
+## Scylla in Docker for the [Mutant Monitoring University Course](https://university.scylladb.com/courses/the-mutant-monitoring-system-training-course/) - Scala Lessons 1 2 and 3
+
+### Instructions for setting up a Scylla Cluster from this repo, prerequisites are a running 3 node cluster with the catalog/tracking keyspaces and tables. More info in the Scylla University course [Using Scylla Drivers](https://university.scylladb.com/courses/using-scylla-drivers/). 
+
+```
+cd mms/scala/
+docker build -t scala-app .
+docker run -d --net=mms_web --name some-scala-app scala-app
+docker exec -it some-scala-app sh
+java -jar scala-app/target/scala-2.13/mms-scala-app-assembly-0.1.0-SNAPSHOT.jar
+```
+
+### To manually add the catalog keyspace and data
+docker exec -it mms_scylla-node1_1 cqlsh
```

Comment:
Triple ticks (```)?

Reply:
Standard markdown
```
Standard markdown
```

Try it for yourself in the following comment ;)